applications/crossbar/doc/phone_numbers.md

```diff
@@ -310,6 +319,85 @@ Possible reasons for failure:
 }
 ```
 
+#### Add a number to the database
```

Comment:
https://github.com/2600hz/kazoo/blob/knm-REST/applications/crossbar/doc/ref/phone_numbers.md#create

Also, can you update any uses of `_` in `allowed_methods/X` to be `_PhoneNumber` or `_WhateverThisIs` so the ref doc generates more meaningful URLs?